pdf/creator/table.go

```diff
@@ -570,8 +570,18 @@ type TableCell struct {
 	table *Table
 }
 
-// NewCell makes a new cell and inserts into the table at current position in the table.
+// NewCell makes a new cell and inserts it into the table at the current position.
 func (table *Table) NewCell() *TableCell {
+	return table.newCell(1)
+}
+
+// MultiColCell makes a new cell with the specified column span and inserts it
+// into the table at the current position.
+func (table *Table) MultiColCell(colspan int) *TableCell {
```

Comment:
Prefer this over a basic SetColSpan function on the cell?  (similar to html styling)

Reply:
The column span should be set from the creation of the cell. Otherwise, each time a cell's column span changes, all the positions of the cells after it, until the end of the table, must be recalculated. Also, there could be cells that are skipped in the table (table.SkipCells) which must be taken into account and this information is not stored anywhere. That method just offsets the position of the next cell upon creation.

Reply:
OK makes sense